.github/workflows/breaking-change-cross-version-code.yaml

```diff
@@ -0,0 +1,70 @@
+name: "[TEST-IGNORE] Breaking Change(Cross-Version) - Analyze Code"
+
+on: pull_request
+
+permissions:
+  contents: read
+
+jobs:
+  validateBreakingChange:
+    name: "[TEST-IGNORE] Breaking Change(Cross-Version) - Analyze Code"
+    runs-on: ubuntu-24.04
+
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
```

Comment:
Are you sure you need `fetch-depth: 0`?  It's OK if you do, but faster if you can use something lower, maybe `2` if the default `1` isn't enough?